spec/acceptance/suites/default/redis_adminstration_spec.rb

```diff
@@ -1,28 +1,30 @@
 require 'spec_helper_acceptance'
 
+RSpec::Matchers.define_negated_matcher :execute_without_warning, :execute_with_warning
+
 # systcl settings are untestable in docker
 describe 'redis::administration', unless: default['hypervisor'] =~ %r{docker} do
-  it 'runs successfully' do
-    pp = <<-EOS
-    include redis
-    include redis::administration
-    EOS
+  def execute_with_warning
+    have_attributes(stderr: %r{WARNING})
+  end
```

Comment:
This here is to combine things with `.and` since you can't use `.not_to().and()`.